### PR TITLE
chore: update kubernetes.md, bumping stable from v2.9.4 to v2.10.2

### PR DESCRIPTION
**Pull request by jmshoffs0812**
Bumped stable version from v2.9.4 to v2.10.2

### CODE REVIEW

**Comment by jmshoffs0812** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jmshoffs0812** [comment]
recheck

**Review by kylecarbs — APPROVED** [review]

**Comment by jmshoffs0812** [comment]
recheck

File: docs/install/kubernetes.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ locally in order to log in and manage templates.
    helm install coder coder-v2/coder \
        --namespace coder \
        --values values.yaml \
-       --version 2.9.4
+       --version 2.10.2
```
Comment (matifali) [inline docs/install/kubernetes.md:148]: @stirby another place to automate with the release script.

Comment (stirby) [reply docs/install/kubernetes.md:148]: good catch

**Pull request merged**